### DISCUSSION TITLE
Network overhead

### DISCUSSION THREAD

**Discussion by gohar94** [TechEmpower/FrameworkBenchmarks#7186] [General]
Hi. I am trying to understand the latency results (e.g., when using Plaintext on Lithium server). What is the client/server network overhead? e.g., when the client sends a ping to the server, what is the latency?

**Comment by bhauer** [comment]
The raw network latency is not measured as a separate metric. However, in the physical test environment ("Citrine"), the test is run on a 10-gigabit Ethernet switch that is dedicated to the project and handles no unrelated traffic. As a result, I expect the latency to be low enough that the `ping` tool would consistently show `0ms`.
